pyteal/ast/router.py

```diff
@@ -637,21 +620,28 @@ def method(
             opt_in (optional): The allowed calls during :code:`OnComplete.OptIn`.
             close_out (optional): The allowed calls during :code:`OnComplete.CloseOut`.
             clear_state (optional): The allowed calls during :code:`OnComplete.ClearState`.
+                This argument has been deprecated, and will error on compile time if one wants to access it.
```

Comment:
Can you mention that they should use the `clear_state` argument when creating a Router here as well?

Reply:
good catch, just add a line for referring using `clear_state` argument.